jina/clients/base/grpc.py

```diff
@@ -138,29 +145,31 @@ def _result_handler(resp):
             result_handler=_result_handler,
             iterate_sync_in_thread=False,
             logger=self.logger,
+            prefetch=prefetch or 0,
```

Comment:
I believe `Streamer` is ready to handle `None` so no need of `or 0`

Reply:
I believe it should be `preferch` or `self.args.prefetch` or `self.prefetch` for backwards compatibility?